common/type_to_string.c

```diff
@@ -5,6 +5,7 @@
 #include <common/type_to_string.h>
 #include <common/utils.h>
 #include <inttypes.h>
+#include <assert.h>
```

Comment:
This failed `make check-source` because we sort includes into alphabetical order to simplify merging.